less/simplesearch.less

```diff
@@ -25,14 +25,17 @@
   }
 }
 
-.view-details-associations, .route-associations, .associations, .associated-documents {
+.view-details-associations, .route-associations, .associations, .associated-documents, #participants-group {
   .tt-open {
     max-height: 400px;
     overflow-y: scroll;
     @media @phone {
       max-height: 300px;
     }
   }
+  app-search {
```

Comment:
Don't you need to make this change since the element is now ``<app-simple-search>`` instead of ``<app-search>``?